sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/OrderStatusResponse.java

```diff
@@ -0,0 +1,71 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import kong.unirest.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+import java.util.Spliterator;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.utils.CalendarUtils.stringToCalendar;
+
+public class OrderStatusResponse {
+    private final String merchantOrderId;
+    private final String id;
+    private final String poiId;
+    private final String orderStatus;
+    private final String statusLastUpdatedAt;
+    private final Money totalAmount;
+    private final List<TransactionInfoOrderStatus> TransactionInfoOrderStatus;
+
+    public OrderStatusResponse(JSONObject jsonObject) {
+        JSONObject orderObject = jsonObject.getJSONObject("element");
+
+        this.merchantOrderId = orderObject.getString("merchantOrderId");
+        this.id = orderObject.getString("id");
+        this.poiId = orderObject.getString("poiId");
+        this.orderStatus = orderObject.getString("orderStatus");
+        this.statusLastUpdatedAt = orderObject.getString("statusLastUpdatedAt");
```

Comment:
statusLastUpdatedAt needs to be removed for now, we have a separate ticket in which we need to investigate how to handle some specific status change scenarios. As soon we decide how to deal with it, we will include this filed back.

Reply:
Removed.